packages/cli/src/commands/ui/create/react.ts

```diff
@@ -97,4 +104,65 @@ export default class React extends Command {
 
     return await platformClient.user.get();
   }
+
+  private async checkIfUserHasNodeGreaterThan10() {
+    const stdio = await spawnProcessStdio('node', ['--version']);
+    if (stdio.stderr && stdio.stderr.match(/ENOENT/i)) {
+      this.warn(`${this.id} requires Node.js to run.`);
+      this.warnHowToInstallNode();
+      return false;
+    }
+
+    if (stdio.stderr) {
+      this.warn(`${this.id} requires a valid Node.js installation to run.`);
+      this.warn(
+        'An unknown error happened while trying to determine your node version with node --version.'
+      );
+      this.warn(stdio.stderr);
+      this.warnHowToInstallNode();
+      return false;
+    }
+
+    const requiredNodeVersionForCreateReactApp = '10.16.0';
+    if (lt(stdio.stdout, requiredNodeVersionForCreateReactApp)) {
+      this.warn(
+        `${this.id} uses create-react-app, which needs a Node.js version greater than ${requiredNodeVersionForCreateReactApp}`
+      );
+      this.warn(`Version detected: ${stdio.stdout}`);
+      this.warnHowToInstallNode();
+      return false;
+    }
+
+    return true;
+  }
+
+  private async checkIfUserHasNpx() {
+    const stdio = await spawnProcessStdio('npx', ['--version']);
+
+    if (stdio.stderr && stdio.stderr.match(/ENOENT/i)) {
+      this.warn(`${this.id} requires npx to run.`);
+      this.warn('Newer version Node.js comes bundled with npx.');
+      this.warnHowToInstallNode();
+      return false;
+    }
+
+    if (stdio.stderr) {
```

Comment:
npx depends on [update-notifier](https://github.com/npm/npx/blob/latest/package.json#L42).
What would happen if the user has an older version of npx and gets the update warning message? Will it be redirected to the stderr?